viva_las_vegas_script_tests.py

```diff
@@ -462,6 +462,133 @@ def test_sending_request_and_rejecting_it_without_permission(self):
         self.switch_channel('leave-coordination')
         self._reject_request(username=self.test_username)
 
+    def test_sending_ill_request_without_working_from_home_and_rejecting_it(self):
+        """Tests if it's possible to send an ill request without working home. """
```

Comment:
Tests if it's not possible to send a partially completed ill request (without working from home).